### ISSUE TITLE
fromLazyPromise is undefined in server-side node

### ISSUE THREAD

**Issue by mbassan** [redux-loop/redux-loop#123]
Here is my code:

```
import {createStore,Effects} from 'redux-loop';
import {Main, INITIAL_STATE} from './lib/Main';
import Connections from './lib/Connections';

export function reducer(state = INITIAL_STATE, action) {
	switch (action.type) {
		case 'MAIN_GET_CFG':
			return Main.getCfg(state);
		case 'MAIN_READY':
			return Main.ready(state,action.value);
		case 'MAIN_SUBSCRIBE':
			return Main.subscribe(state);
		case 'CONN_INIT':
			return Connections.init(state);
		case 'CONN_SUBSCRIBE':
			return Connections.subscribe(state);
		case 'CONN_GET_PRICE':
			return Connections.getPrice(state);
		case 'CONN_SET_PRICE':
			return Connections.setPrice(state,action.id,action.value);
		case 'CONN_GET_DEPTH':
			return Connections.getDepth(state);
		case 'CONN_SET_DEPTH':
			return Connections.setDepth(state,action.id,action.value);
		default:
			return state
	}
}

export const init = {
	state: INITIAL_STATE,
	effects: [ Effects.fromLazyPromise(() => Promise.resolve({ type: 'MAIN_GET_CFG' })) ]
};
```

And here is the error I'm getting:

TypeError: _reduxLoop.Effects.fromLazyPromise is not a function
    at Object.<anonymous> (/var/www/project/src/reducer.js:32:21)
    at Module._compile (module.js:571:32)
    at loader (/var/www/project/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/var/www/project/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/var/www/project/src/store.js:4:1)

Simply included it in the npm package.json and did npm install. I'm using node 7.4.0 Any ideas?

**Comment by MichalZalecki** [comment]
Are you using `2.2.2`? What's your build process?

**Comment by bdwain** [comment]
hi @mbassan and @MichalZalecki 

Sorry it's been so long since you've heard back. I'm going to be taking over maintenance of redux-loop.

fromLazyPromise is from a version of redux-loop that was never released. The current documentation is not in sync with the latest npm version. I'd like to get an update out in the next few days that will get them back in sync and up to date.

There have been a few proposed major API changes in the last year. I've been using a fork of one those proposed version 3's for a while in projects that I've worked on (https://github.com/bdwain/redux-loop). I'd like to make that the starting point for any future changes. Luke and I discussed these changes last year and I think it'll be an improvement over version 2 without being too drastic of a change for people who are currently using it. It doesn't have typescript typings yet, but I would like to add that in the next few days as well.

The main idea behind the changes in my branch can be seen here [in this issue](https://github.com/redux-loop/redux-loop/issues/66) . It's passing action creators to effects/cmds instead of putting them in your side effect functions. Though since that discussion, i've consolidated some of the Cmd types.

I'll keep this issue open for now and close it when I've got everything updated. Sorry again for the delay!
